### PR TITLE
Updated timezone choices

### PR DESCRIPTION
**Pull request by aarthi-axim**
## Status
**READY**

## Description
Updated the user timezone choices.

## Trello Ticket(s)
https://trello.com/c/ngUFq7PT

## Screenshots and Screen Captures (if appropriate):
narrow_timezones

### CODE REVIEW

File: apps/accounts/forms/update.py [hunk 1]
```diff
@@ -146,7 +147,10 @@ def __init__(self, instance, *args, **kwargs):
         """Initialization for TimeZoneForm.
         """
         user_profile = instance
+        timezone = [user_profile.timezone for tz in US_TIMEZONE_CHOICES if user_profile.timezone in tz]
+        user_profile.timezone = timezone[0] if timezone else htk_setting('HTK_DEFAULT_TIMEZONE', 'UTC')
         super(TimeZoneForm, self).__init__(user_profile, *args, **kwargs)
+        self.fields['timezone'].choices = [(x, y,) for x, y in US_TIMEZONE_CHOICES]
```
Comment (jontsai) [inline apps/accounts/forms/update.py:153]: Don't hardcode the timezone choices here

Instead, pass it in `**kwargs`

```
timezone_choices = kwargs.get('timezone_choices', None)
if timezone_choices:
    self.fields['timezone'].choices = [('', '', ), ] + timezone_choices
```

(include a blank choice)

File: apps/accounts/forms/update.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
 # HTK Imports
 from htk.apps.accounts.emails import password_changed_email
+from htk.constants.i18n.timezones import US_TIMEZONE_CHOICES
```
Comment (jontsai) [inline apps/accounts/forms/update.py:15]: remove, this should be passed in from the call site that initializes this form

File: apps/accounts/forms/update.py [hunk 3]
```diff
@@ -146,7 +147,10 @@ def __init__(self, instance, *args, **kwargs):
         """Initialization for TimeZoneForm.
         """
         user_profile = instance
+        timezone = [user_profile.timezone for tz in US_TIMEZONE_CHOICES if user_profile.timezone in tz]
+        user_profile.timezone = timezone[0] if timezone else htk_setting('HTK_DEFAULT_TIMEZONE', 'UTC')
```
Comment (jontsai) [inline apps/accounts/forms/update.py:151]: I'm not sure that these two lines are needed. Could you explain why? Please also see the comment below, and address that first, and see if it changes anything

**Review by jontsai — CHANGES_REQUESTED** [review]

File: apps/accounts/forms/update.py (hunk 1 above)
Comment (aarthi-axim) [reply apps/accounts/forms/update.py]: Done

File: apps/accounts/forms/update.py (hunk 2 above)
Comment (aarthi-axim) [reply apps/accounts/forms/update.py]: Done

File: apps/accounts/forms/update.py (hunk 3 above)
Comment (aarthi-axim) [reply apps/accounts/forms/update.py]: Removed the added lines.

**Review by jontsai — APPROVED** [review]

**Pull request merged**